test/salt/pillar/default.sls

```diff
@@ -93,7 +93,7 @@ prometheus:
         archive:
           source_hash: b2503fd932f85f4e5baf161268854bf5d22001869b84f00fd2d1f57b51b72424
         service:
-          # name: prometheus-node-exporter
+          name: node_exporter
```

Comment:
This is useless because the formula fallbacks to `node_exporter`

Reply:
This proves nothing whereas setting to `prometheus-node-exporter` proves that in archive mode the service name is wrong and should not be added.

Reply:
The formula falls back to `node_exporter` if no `osfamily.yaml` `service:name`s are added. But as I mention above, I set the defaults to be consistent in the debian family so, in this case, if we want to *test* that the name is *node_exporter* in debian, we need to explicitely set it in the pillar.

OTOH, if we want to test the defaults the formula sets now, I can change it to be a conditional test on the family, testing for `node_exporter` in non-Debian or `prometheus-node-exporter` on Debian

Reply:
> I set the defaults to be consistent in the debian family

It is consistent in the debian family if use the repo mode not the archive mode.

Reply:
even more useless :)

Reply:
you're right! removed.